src/components/with_search_destenation.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+export default Component => ({ showRightSearch, selectedItems, ...others }) => {
+  class App extends React.Component {
+    constructor(props) {
+      super(props);
+
+      this.state = {
+        searchValue: ""
```

Comment:
i think we already talked about this. we need to do the same approach as in existing search. why you added this?